src/js/components/AutoScrollControl/AutoScrollControl.tsx

```diff
@@ -97,74 +110,75 @@ class AutoScrollControl extends React.PureComponent<IAutoScrollControlProps, IAu
     this.props.setAutoScrolling && this.props.setAutoScrolling(true);
-    this._nextScrollPosition = document.documentElement.scrollTop;
+    this._nextScrollPosition =
+      window.pageYOffset || document.documentElement.scrollTop;
     this._setMaxScrollPossible();
     this._interval = requestAnimationFrame(this.handleAutoScroll);
-  }
+  };
 
   handleScrollbarClick = (e: MouseEvent) => {
-    const isScrollbarClicked = window.innerWidth >= e.pageX && e.pageX >= document.body.scrollWidth;
+    const isScrollbarClicked =
+      window.innerWidth >= e.pageX && e.pageX >= document.body.scrollWidth;
     if (isScrollbarClicked) {
+      e.preventDefault();
+      e.stopPropagation();
       this.removeScroll();
     }
-  }
+  };
 
   removeScroll = () => {
     this.props.setAutoScrolling && this.props.setAutoScrolling(false);
-    this.clearScrollInterval()
-  }
+    this.clearScrollInterval();
+  };
 
   _setMaxScrollPossible = () => {
-    this._maxScrollPossible = document.documentElement.scrollHeight - window.innerHeight;
-  }
+    this._maxScrollPossible =
+      document.documentElement.scrollHeight - window.innerHeight;
+  };
 
   handleAutoScroll = () => {
     if (this.props.isAutoScrolling) {
-      const scrollY = document.documentElement.scrollTop;
+      const scrollY = window.pageYOffset || document.documentElement.scrollTop;
+
       if (scrollY >= this._maxScrollPossible && !this.props.isLoadingAng) {
-        // we are resetting the maxscroll value since it can hold old value by now.
         this._setMaxScrollPossible();
-
         const isMaxScrollCrossed = this._maxScrollPossible >= scrollY;
         if (isMaxScrollCrossed) {
           this.removeScroll();
-          if(this.state.autoScrollLoopMode){
-            const timeoutInterval = setTimeout(() => {
-              window.scrollTo(0, 0);
-              setTimeout(() => {
-                this.startScroll();
-                clearTimeout(timeoutInterval);
-              }, 1000);
-            },1000)
+          if (this.state.autoScrollLoopMode) {
+            window.scrollTo(0, 0);
+            setTimeout(() => {
+              this.startScroll();
+            }, 1000);
           }
         }
       }
 
       const [scrollingSpeed] = this.state.scrollingSpeed;
+      const movement = toFixedFloat(
+        0.1 + (scrollingSpeed / 100) * AutoScrollControl.thresHold,
+        2
+      );
 
-      // We are having minimum scroll per pixel + adding the extra dynamic pixel movement based on slider value.
-      const movement = toFixedFloat((0.1 + ((scrollingSpeed / 100) * AutoScrollControl.thresHold)), 2);
-      // Only allow the scrolling if we have surpassed previous scrolls or if it's firefox browser.
-      if (this._isFirefoxAgent || scrollY >= Math.round(this._nextScrollPosition)) {
-        this._nextScrollPosition += movement;
-        window.scrollTo({ left: 0, top: Math.round(this._nextScrollPosition)});
-      }
+      // Update next position and scroll
```

Comment:
@Gauravjeetsingh In the previous logic, we had an if condition to prevent scrollY updates when the new scroll position surpassed the previous one. I've reintroduced that check to avoid unnecessary scroll position updates.
It's a small detail, but it's good to keep track of these changes for consistency.

Reply:
Fixed, added that condition back